decidim-core/spec/commands/decidim/search_spec.rb

```diff
@@ -8,7 +8,7 @@
   context "with resources from different organizations" do
     let(:other_organization) { create(:organization) }
     let(:term) { "fire" }
-    let(:fake_type) { "Decidim::DoesNot::Exist" }
+    let(:fake_type) { "Decidim::Organization" } # a type that is not searchable
```

Comment:
Without this we have this error:

https://github.com/decidim/decidim/actions/runs/8342208106/job/22829977839

```
  1) Decidim::Search with resources from different organizations only returns searchable results
     Failure/Error: create(:searchable_resource, organization: current_organization, resource_type: fake_type, content_a: "Where is your crown king nothing?")

     NameError:
       uninitialized constant Decidim::DoesNot
     # ./spec/commands/decidim/search_spec.rb:16:in `block (3 levels) in <top (required)>'
```

I think that the problem is that you can't create a Decidim::SearchableResource with a non-existing resource type any more, so I think it's better to change this to an invalid class (that doesn't implement Searchable) 

Basically, this is the issue: 

```
> bin/rails runner 'Decidim::SearchableResource.create(organization: Decidim::Organization.first, resource: Decidim::Proposals::Proposal.first, resource_type: "Decidim::DoesNot::Exist")'
Running via Spring preloader in process 3024478
Please specify a valid ruby command or the path of a script to run.
Run 'rails runner -h' for help.

uninitialized constant Decidim::DoesNot
```